apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -80,6 +80,7 @@ export const BaseWorkflowByIdSchema = z.object({
   assignee: ObjectWithIdSchema.extend({
     firstName: z.string(),
     lastName: z.string(),
+    avatarUrl: z.string().nullable().optional(),
```

Comment:
Main change 3/3 - now using `avatarUrl` from `workflow.assignee` instead of `entity`, until now the avatar in the dropdown was broken.